src/validation.cpp

```diff
@@ -3702,7 +3714,7 @@ bool RewindBlockIndex(const CChainParams& params)
 {
     LOCK(cs_main);
 
-    int nHeight = 1;
+    int nHeight = chainActive.Height();
```

Comment:
having `height = 1` resulted in rewinding the block index and then requiring redownloading/reprocessing the blockchain.